java/flight/flight-core/src/test/java/org/apache/arrow/flight/auth/TestBasicAuth.java

```diff
@@ -97,7 +96,12 @@ public void didntAuth() {
   }
 
   @BeforeEach
-  public void setup() throws IOException {
+  public void testSetup() throws IOException {
```

Comment:
These test utilize stateful authentication so a new FlightClient must be created for each test.